app/main/views/suppliers.py

```diff
@@ -444,3 +443,28 @@ def create_your_account_complete():
         "suppliers/create_your_account_complete.html",
         email_address=email_address
     ), 200
+
+
+@main.route('/<framework_slug>/opportunities-overview', methods=['GET'])
```

Comment:
And if all this stuff is going to be moving to the new briefs app then `briefs.py` might be the natural home for it for now, as I assume those are the views being ported across.

I'm not sure how we're going to handle the links from the dashboard though once apps are split - we're going to lose the ability to use `url_for` and probably end up hard-coding routes, which is a real shame.